packages/mux-uploader-react/README.md

```diff
@@ -48,8 +48,8 @@ const MuxUploaderWithMuxUploaderDropExample = () => {
   return (
     <div>
       <h1>Simple MuxUploader and Mux Uploader Drop Examples</h1>
-      {/* Upload button by itself. Displays upload progress in text as percentage. */}
-      <MuxUploader url="authenticated-url" type="bar" status></MuxUploader>
+      {/* Rounded upload button by itself. Displays upload progress in text as percentage. */}
+      <MuxUploader url="authenticated-url" type="bar" status style={{ '--button-border-radius': '40px' }}></MuxUploader>
```

Comment:
is this still right `url` vs `endpoint`? 

why was this renamed, I don't see any reference to endpoint in the Mux video docs to `endpoint`

Reply:
Out of date. Will fix.